Code/Blueprint.cs

```diff
@@ -8,18 +8,16 @@ public partial class Blueprint : Button
     [Export]
     private String _title;
 
-    [Export]
-    private Array<Materials> _materials;
-
-    [Export]
-    private Array<int> _quantities;
+    public Dictionary<Materials, int> _recipe;
```

Comment:
why have it as recipe instead of blueprint. just adding 1 more term to remember what I'm trying to get at

Reply:
https://github.com/adrian-azan/Valheim-Todo/pull/3/commits/9fd76698a25a896a32d713b2ea724d4ada5837b6